addon/components/detail.js

```diff
@@ -588,8 +592,14 @@ export default Component.extend(SpreadMixin, HookMixin, PropTypeMixin, {
         component.unregisterForFormValueChanges(component)
       })
 
+      const formValue = this.get('renderValue') || {}
+
       // Make sure we inform component of formValue immediately
-      component.formValueChanged(this.get('renderValue') || {})
+      if (typeOf(component.formValueChanged) === 'function') {
```

Comment:
shouldn't it be `get('formValueChanged')` either way?  Or is the intent here to respect a hardwired `formValueChange` method on the component?  What if `formValueChanged` is passed, but as a function, not a task?